app/views/leaderboard.scala.html

```diff
@@ -27,9 +27,19 @@ <h5 class="leaderboard-detail">@Messages("leaderboard.overall.detail")</h5>
                         <th class="leaderboard-table-font" scope="row">@{
                             i + 1
                         }</th>
-                        <td class="leaderboard-table-font">@{
-                            stat.asInstanceOf[LeaderboardStat].username
-                        }</td>
+                        @if(user && user.get.role.getOrElse("") != "Anonymous" && stat.asInstanceOf[LeaderboardStat].username == user.get.username){
```

Comment:
Add space between `)` and `{` as per [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide)